### DISCUSSION TITLE
开源嘉年华纪实 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#130] [Comments]
# 开源嘉年华纪实 | 夜天之书

上周在北京参与了开源社主办的 2024 中国开源年会。其实相比于有点明显班味的“年会”，我的参会体验更像是经历了一场中国开源的年度嘉年华。这也是在会场和其他参会朋友交流时共同的体验：在开源社的 COSCon 活动上，能够最大限度地一次性见到所有在中国从事开源相关工作的人。 我在本次会场上有两个演讲，分别是 Rust 分论坛上分享的《高性能 Rust 编程：如何减少数据拷贝和并发开销》，以及开源运营

https://www.tisonkun.org/2024/11/06/open-source-carnival/

**Comment by psxjoy** [comment]
Hi，请问您在[主题分享：高性能-Rust](https://www.tisonkun.org/2024/11/06/open-source-carnival/#%E4%B8%BB%E9%A2%98%E5%88%86%E4%BA%AB%EF%BC%9A%E9%AB%98%E6%80%A7%E8%83%BD-Rust)这个模块中，统计自己码力值，是从哪里生成的呢？能否分享一下？谢谢

**Reply by tisonkun** [reply]
https://ossinsight.io/analyze-user/tisonkun#overview
